api/files/views.py

```diff
@@ -36,10 +35,10 @@ def get_file(self, check_permissions=True):
 class FileDetail(generics.RetrieveUpdateAPIView, FileMixin):
     """Details about a specific file.
     """
+    __metaclass__ = OsfAPIViewMeta
+
     permission_classes = (
-        drf_permissions.IsAuthenticatedOrReadOnly,
         CheckedOutOrAdmin,
-        base_permissions.TokenHasScope,
```

Comment:
True, but if they took that off (as an experiment, for example) and it didn't change anything, it would be weird in a different way.